content/docs/contributing/release-process.md

```diff
@@ -308,7 +308,18 @@ page if a step is missing or if it is outdated.
         only to avoid rate-limits imposed on anonymous API users.
         </p></div>
 
-    3. Sanity check the notes:
+    3. Replace the GitHub issue numbers and GitHub handles (e.g., `#1234` or
```

Comment:
I guess this only needs to be done for the version of release notes that are going to be committed here- should we add a note about that?

Reply:
I think that it's the other way round. I meant adding some note to the sed script that explains that this is only needed to be done when preparing the release note for website.

Reply:
You are totally right! Thanks.